recipe/meta.yaml

```diff
@@ -58,7 +58,7 @@ requirements:
     - cirun >=0.30
     - exceptiongroup
     - jsonschema
-    - rattler-build-conda-compat >=0.2.4
+    - rattler-build-conda-compat >=0.2.4,<1.0.0a0
```

Comment:
Do we need a repodata patch too?

Reply:
Ah looks like that is in PR: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/812